shenyu-sync-data-center/shenyu-sync-data-http/src/main/java/org/apache/shenyu/sync/data/http/refresh/DataRefreshFactory.java

```diff
@@ -57,9 +57,9 @@ public DataRefreshFactory(final PluginDataSubscriber pluginDataSubscriber,
      * @return the boolean
      */
     public boolean executor(final JsonObject data) {
-        final boolean[] success = {false};
-        ENUM_MAP.values().parallelStream().forEach(dataRefresh -> success[0] = dataRefresh.refresh(data));
-        return success[0];
+        return ENUM_MAP.values().parallelStream()
+                .map(dataRefresh -> dataRefresh.refresh(data))
+                .anyMatch(Boolean.TRUE::equals);
```

Comment:
This causes some values of `ENUM_MAP` to return without executing the `refresh ` method. I don't think we should take this PR.